src/status_im/ui/screens/profile/visibility_status/utils.cljs

```diff
@@ -91,27 +88,17 @@
 
 (defn icon-dot-accessibility-label
   [dot-color]
-  (if @config/new-ui-enabled?
-    (if (= dot-color quo2.colors/success-50)
-      :online-profile-photo-dot
-      :offline-profile-photo-dot)
-    (if (= dot-color colors/color-online)
-      :online-profile-photo-dot
-      :offline-profile-photo-dot)))
+  (if (= dot-color quo2.colors/success-50)
+    :online-profile-photo-dot
+    :offline-profile-photo-dot))
 
 (defn icon-dot-margin
-  [size identicon?]
-  (if @config/new-ui-enabled?
-    -2
-    (if identicon?
-      (/ size 6)
-      (/ size 7))))
+  [_ _]
+  -2)
```

Comment:
So even if we opt for old ui, these params will be still used as per new ui?

Reply:
yes, because we don't want to support old ui anymore, this one was tricky because it somehow was implemented without new design, so because we won't support old ui anymore i decided to keep only new ui even if it was not properly implemented

Reply:
we shouldn't change old ui anymore